models/shopify.yml

```diff
@@ -971,7 +971,11 @@ models:
       Each record represents a unique discount, enriched with discount metadata and metrics regarding orders and abandoned checkouts.
     columns:
       - name: discounts_unique_key
-        description: Unique key representing a discount. Hashed on 'discount_code_id' and 'source_relation'.
+        description: |
+          Unique key representing a discount. Hashed on base: 'source_relation', 'discount_code_id';
+          stg_shopify__discount_redeem_code: 'discount_code_id', 'discount_id', 'discount_type';
+          stg_shopify__discount_application: 'code', 'allocation_method', 'description', 'target_selection',
+          'target_type', 'type', 'value', 'value_type'.
```

Comment:
This format is a little confusing to me... Since all of the individual fields are included in the model, and the overlapping ones like `discount_code_id` were included in joins, perhaps we should just provide a unique list of fields hashed on (ie leave the originating models out)